src/ucp/api/ucp.h

```diff
@@ -582,16 +582,22 @@ enum ucp_am_cb_flags {
  * routines.
  */
 enum ucp_send_am_flags {
-    UCP_AM_SEND_FLAG_REPLY = UCS_BIT(0),             /**< Force relevant reply
-                                                          endpoint to be passed to
-                                                          the data callback on the
-                                                          receiver. */
-    UCP_AM_SEND_FLAG_EAGER = UCS_BIT(1),             /**< Force UCP to use only
-                                                          eager protocol for AM sends. */
-    UCP_AM_SEND_FLAG_RNDV  = UCS_BIT(2),             /**< Force UCP to use only
-                                                          rendezvous protocol for
-                                                          AM sends. */
-    UCP_AM_SEND_REPLY      = UCP_AM_SEND_FLAG_REPLY  /**< Backward compatibility. */
+    UCP_AM_SEND_FLAG_REPLY       = UCS_BIT(0),             /**< Force relevant reply
+                                                                endpoint to be passed to
+                                                                the data callback on the
+                                                                receiver. */
+    UCP_AM_SEND_FLAG_EAGER       = UCS_BIT(1),             /**< Force UCP to use only
+                                                                eager protocol for AM sends. */
+    UCP_AM_SEND_FLAG_RNDV        = UCS_BIT(2),             /**< Force UCP to use only
+                                                                rendezvous protocol for
+                                                                AM sends. */
+    UCP_AM_SEND_FLAG_COPY_HEADER = UCS_BIT(3),             /**< The user header buffer is temporary
+                                                                and is valid only within the scope
+                                                                of this function call.
+                                                                If needed by UCX for later use, 
```

Comment:
@tonycurtis, I suggest:
`The flag indicates that the UCX has to copy the header in order to access it after function call return. Once the function call returns, user car reuse or release the header.` What do you prefer.

Reply:
Sounds ok to me (car -> can)

Reply:
`the UCX` -> `UCX`